lib/pages/chat_list/chat_list_item_style.dart

```diff
@@ -27,10 +27,11 @@ class ChatListItemStyle {
         : 0.0;
   }
 
-  static const EdgeInsetsDirectional paddingConversation =
-      EdgeInsetsDirectional.symmetric(
-    horizontal: 8,
-    vertical: 2,
+  static const EdgeInsets paddingConversation = EdgeInsets.fromLTRB(
+    8,
+    8,
+    8,
+    0,
```

Comment:
Wrong padding for chat list item
![telegram-cloud-photo-size-5-6271625590989766557-y](https://github.com/user-attachments/assets/6d7e07c6-5a1c-468c-8de7-5a3beac3313b)

Reply:
- [ ] avatar must be in the center of chat list item